test/functional/apps/dashboard/dashboard_filter_bar.js

```diff
@@ -67,7 +67,7 @@ export default function ({ getService, getPageObjects }) {
 
       it('uses default index pattern on an empty dashboard', async () => {
         await testSubjects.click('addFilter');
-        await dashboardExpect.fieldSuggestions(['bytes']);
+        await dashboardExpect.fieldSuggestions(['agent']);
```

Comment:
> Changed underlying list virtualization library. Some EuiComboBox and EuiFilter* lists will technically have one fewer item in the DOM on initial load, although no real change occurs because the now-absent item was not visible initially without scrolling.